assets/quick-start-module.js

```diff
@@ -174,265 +174,211 @@ $("[data-toggle='cloud-dropdown']").on("click", function(e) {
-    "stable,conda,macos,cudanone,python":
+    "stable,conda,macos,accnone,python":
       "<b>NOTE:</b> Python 3.9 users will need to add '-c=conda-forge' for installation<br />conda install pytorch torchvision torchaudio -c pytorch",
 
-    "stable,conda,windows,cuda10.1,python":
+    "stable,conda,windows,cuda11.1,python":
       "<b>NOTE:</b> Python 3.9 users will need to add '-c=conda-forge' for installation<br />conda install pytorch torchvision torchaudio cudatoolkit=10.1 -c pytorch",
 
-    "stable,conda,windows,cuda10.2,python":
+    "stable,conda,windows,rocm4.0,python":
       "<b>NOTE:</b> Python 3.9 users will need to add '-c=conda-forge' for installation<br />conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch",
 
-    "stable,conda,windows,cuda11.0,python":
-      "<b>NOTE:</b> Python 3.9 users will need to add '-c=conda-forge' for installation<br />conda install pytorch torchvision torchaudio cudatoolkit=11.0 -c pytorch",
-
-    "stable,conda,windows,cudanone,python":
+    "stable,conda,windows,accnone,python":
       "<b>NOTE:</b> Python 3.9 users will need to add '-c=conda-forge' for installation<br />conda install pytorch torchvision torchaudio cpuonly -c pytorch",
 
-    "stable,pip,macos,cuda9.2,python":
+    "stable,pip,macos,cuda10.2,python":
       "pip install torch torchvision torchaudio<br /># MacOS Binaries dont support CUDA, install from source if CUDA is needed",
 
-    "stable,pip,macos,cuda10.1,python":
+    "stable,pip,macos,cuda11.1,python":
       "pip install torch torchvision torchaudio<br /># MacOS Binaries dont support CUDA, install from source if CUDA is needed",
 
-    "stable,pip,macos,cuda10.2,python":
+    "stable,pip,macos,rocm4.0,python":
       "pip install torch torchvision torchaudio<br /># MacOS Binaries dont support CUDA, install from source if CUDA is needed",
 
-    "stable,pip,macos,cuda11.0,python":
-      "pip install torch torchvision torchaudio<br /># MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+    "stable,pip,macos,accnone,python": "pip install torch torchvision torchaudio",
 
-    "stable,pip,macos,cudanone,python": "pip install torch torchvision torchaudio",
-
-    "stable,pip,linux,cudanone,python":
+    "stable,pip,linux,accnone,python":
       "pip install torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html",
 
-    "stable,pip,linux,cuda9.2,python":
+    "stable,pip,linux,cuda10.2,python":
       "pip install torch==1.7.1+cu92 torchvision==0.8.2+cu92 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html",
 
-    "stable,pip,linux,cuda10.1,python":
+    "stable,pip,linux,cuda11.1,python":
       "pip install torch==1.7.1+cu101 torchvision==0.8.2+cu101 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html",
 
-    "stable,pip,linux,cuda10.2,python":
+    "stable,pip,linux,rocm4.0,python":
       "pip install torch torchvision torchaudio",
```

Comment:
```suggestion
      "pip install -f https://download.pytorch.org/whl/rocm4.0/torch_stable.html torch",
```

Currently `rocm` only supports `torch`